.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Docs Build
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  build-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Generate Docs
+        run: |
+          npm i --include=dev
```

Comment:
Is it beneficial to use `npm ci` here?

Reply:
I don't see any reason why it wouldn't be, I'll make the change now.

Reply:
Awesome 👍